worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -0,0 +1,282 @@
+package migrations
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/cozy/cozy-stack/model/app"
+	"github.com/cozy/cozy-stack/model/bitwarden"
+	"github.com/cozy/cozy-stack/model/bitwarden/settings"
+	"github.com/cozy/cozy-stack/model/instance"
+	"github.com/cozy/cozy-stack/model/job"
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/crypto"
+	"github.com/cozy/cozy-stack/pkg/metadata"
+	"github.com/cozy/cozy-stack/web/data"
+	"github.com/sirupsen/logrus"
+
+	multierror "github.com/hashicorp/go-multierror"
+)
+
+type VaultReference struct {
```

Comment:
Does `VaultReference` need to be exported? If yes, please add a comment.

Reply:
It did not have to be public. I switch the case.